docs/docker_container.md

```diff
@@ -256,7 +255,9 @@ It uses `json` format as shown in the example below:
          "config":{
              "name":"model_name5",
              "base_path":"s3://bucket/models/model5",
-             "shape": "auto"
+             "shape": "auto",
+             "target_device": "HDDL",
+             "plugin_config": {"CPU_THROUGHPUT_STREAMS": "CPU_THROUGHPUT_AUTO"}
```

Comment:
I know it's just and example, but to avoid confusion please move one of these parameters to another model's config.
`CPU_*` params won't work with HDDL as a target device.

Reply:
Adding `nireq` would also be cool 😄